BaseFilter/sections/specific.txt

```diff
@@ -326,9 +326,10 @@ lovethemaldives.com##div[class*="-rectangle-"]
 allnovascotia.com##div[style="height:130px;text-align:center;background-color:#eef2f5"]
 tutlehd.xyz###overlayer
 portableapps.com##.ez-sidebar-wall
-check-host.net#?#div + div[class*=" "]:last-child:contains(/^(×|✕|✖|X|☓|χ|x|⨯)$/):upward(2)
-check-host.net###hostign
-/^https:\/\/check-host\.net\/images\/[0-9]?_?[0-9]+[a-z]+(_|[0-9]?[a-z-])+(s?ale|en|ru)?\.png\?[0-9]+$/$domain=check-host.net
+check-host.net#?#div + div[class*=" "]:last-child:contains(/^(χ|Χ|х|𝕏|🗙|🗶|🗴|✘|✗|⨯|☓|×|✕|✖|X|x)$/):upward(2)
```

Comment:
New `x` values better add to the end or at the beginning. No need to add all the values, just the ones they use at the moment.

Reply:
Where are you seeing all these? `check-host.net#?#div + div[class*=" "]:last-child:contains(/^(×|✕|✖|X|☓|χ|x|⨯|✗)$/):upward(2)`  The only one I can find is `✗` . They did, however, use them before, but I can't seem to find anything in their HTML code right now that indicates there still using them.

Reply:
Never mind they just updated now its a `+`